py/selenium/webdriver/remote/errorhandler.py

```diff
@@ -49,87 +50,66 @@
 from selenium.common.exceptions import UnknownMethodException
 from selenium.common.exceptions import WebDriverException
 
-
-class ExceptionMapping:
```

Comment:
By removing this logic we make the client 100% incompatible with the legacy JWP protocol. 

In scope of Appium I can guarantee that the server only supports W3C one and we do not care about JWP support anymore. I cannot guarantee this is the case in Selenium world though. Selenium java client, for example, still supports both JWP and W3C protocols, for example. 

Leaving this decision up to client maintainers.

Reply:
Yes, need feedback at the point. I saw W3C flag removals in the past so I thought this client was removing old protocol behaviors but could be wrong

Reply:
Selenium does not support JWP, but there are still various pieces of JWP-only code left that haven't been removed. If it doesn't apply to w3c we want it gone. :)